core/templates/services/playthrough-issues-backend-api.service.spec.ts

```diff
@@ -50,81 +46,69 @@ describe('PlaythroughIssuesBackendApiService', function() {
+  afterEach(() => {
+    httpTestingController.verify();
   });
 
-  describe('.fetch', function() {
-    it('returns the issues data provided by the backend', function() {
-      var successHandler = jasmine.createSpy('success');
-      var failureHandler = jasmine.createSpy('failure');
-      $httpBackend.expectGET(
-        '/issuesdatahandler/7?exp_version=1'
-      ).respond(backendIssues);
+  describe('.fetch', () => {
+    it('returns the issues data provided by the backend', fakeAsync(() => {
+      let successHandler = jasmine.createSpy('success');
+      let failureHandler = jasmine.createSpy('failure');
 
-      PlaythroughIssuesBackendApiService.fetchIssues('7', 1).then(
+      playthroughIssuesBackendApiService.fetchIssues('7', 1).then(
         successHandler, failureHandler);
-      $httpBackend.flush();
 
+      let req = httpTestingController.expectOne(
+        '/issuesdatahandler/7?exp_version=1');
+      expect(req.request.method).toEqual('GET');
+      req.flush(backendIssues);
+
+      flushMicrotasks();
       expect(successHandler).toHaveBeenCalledWith(
-        backendIssues.map(PlaythroughIssueObjectFactory.createFromBackendDict));
+        backendIssues.map(playthroughIssueObjectFactory.createFromBackendDict));
       expect(failureHandler).not.toHaveBeenCalled();
-    });
+    }));
 
     it('should not fetch an issue when another issue was already fetched',
-      function() {
-        var successHandler = jasmine.createSpy('success');
-        var failureHandler = jasmine.createSpy('failure');
-        $httpBackend.expectGET(
-          '/issuesdatahandler/7?exp_version=1'
-        ).respond(backendIssues);
-
-        PlaythroughIssuesBackendApiService.fetchIssues('7', 1).then(
+      fakeAsync(() => {
+        let successHandler = jasmine.createSpy('success');
+        let failureHandler = jasmine.createSpy('failure');
+
+        playthroughIssuesBackendApiService.fetchIssues('7', 1).then(
           successHandler, failureHandler);
-        $httpBackend.flush();
+
+        let req = httpTestingController.expectOne(
+          '/issuesdatahandler/7?exp_version=1');
+        expect(req.request.method).toEqual('GET');
+        req.flush(backendIssues);
+
+        flushMicrotasks();
 
         expect(successHandler).toHaveBeenCalledWith(
           backendIssues.map(
-            PlaythroughIssueObjectFactory.createFromBackendDict));
+            playthroughIssueObjectFactory.createFromBackendDict));
         expect(failureHandler).not.toHaveBeenCalled();
 
         // Try to fetch another issue
-        PlaythroughIssuesBackendApiService.fetchIssues('8', 1).then(
+        playthroughIssuesBackendApiService.fetchIssues('8', 1).then(
           successHandler, failureHandler);
 
-        expect(successHandler).toHaveBeenCalledWith(
-          backendIssues.map(
-            PlaythroughIssueObjectFactory.createFromBackendDict));
+        expect(successHandler).toHaveBeenCalled();
```

Comment:
Please do not change this to toHaveBeenCalled(). The original piece of code was fine. Revert this back to original.